cli/src/installers/prettier.ts

```diff
@@ -0,0 +1,36 @@
+import fs from "fs-extra";
+import path from "path";
+import { PKG_ROOT } from "~/consts.js";
+import { type Installer } from "~/installers/index.js";
+import { addPackageDependency } from "~/utils/addPackageDependency.js";
+
+export const prettierInstaller: Installer = ({ projectDir, packages }) => {
```

Comment:
i wonder if it'd be a good shout to combine this installer with eslint?

so the default would be no eslint n'or prettier, and you could choose an `eslint+prettier` installer?

Reply:
If you're looking at #1332 as well i think moving eslint & prettier to its own option would be nice. Not everyone wants opinionated formatting or linting for every project. It would also mimick the 'create-next-app' eslint option